src/status_im2/contexts/wallet/events.cljs

```diff
@@ -204,6 +204,7 @@
    (let [request-params [0
                          [(chain/chain-id db)]
                          (map :address (:profile/wallet-accounts db))
+                         nil
```

Comment:
what is `nil` relating to here? perhaps an inline comment would help here?

Reply:
or better a named `def` :)

Reply:
👌  agree, updated

Reply:
same for `0` in the first param slot 👌